### PR TITLE
Test mysql adapter 

### PR DESCRIPTION
**Pull request by fogelito**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you added your change to the [Changelog](https://github.com/appwrite/appwrite/blob/master/CHANGES.md)?

(The CHANGES.md file tracks all the changes that make it to the `main` branch. Add your change to this file in the following format)
- One line description of your PR [#pr_number](Link to your PR)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -48,10 +48,10 @@
         "utopia-php/abuse": "0.14.*",
         "utopia-php/analytics": "0.2.*",
         "utopia-php/audit": "0.15.*",
-        "utopia-php/cache": "0.6.*",
+        "utopia-php/cache": "0.6.0 as 0.7.0",
         "utopia-php/cli": "0.13.*",
         "utopia-php/config": "0.2.*",
-        "utopia-php/database": "0.26.*",
+        "utopia-php/database": "0.27.1 as 0.26.99",
```
Comment (eldadfux) [inline composer.json:54]: Reminder to change this.

**Review by eldadfux — APPROVED** [review]

File: app/config/collections.php [hunk 2]
```diff
@@ -2076,7 +2076,7 @@
                 '$id' => ID::custom('name'),
                 'type' => Database::VAR_STRING,
                 'format' => '',
-                'size' => 2048,
+                'size' => 128,
```
Comment (christyjacob4) [inline app/config/collections.php:2079]: We might need to allow for more characters in the name. 128 may be limiting

File: app/config/collections.php [hunk 3]
```diff
@@ -2424,7 +2424,7 @@
                 '$id' => ID::custom('_key_entrypoint'),
                 'type' => Database::INDEX_KEY,
                 'attributes' => ['entrypoint'],
-                'lengths' => [2048],
+                'lengths' => [768],
```
Comment (christyjacob4) [inline app/config/collections.php:2427]: Hmm.. These seem like magic numbers for anyone referring to it in the future. 
Can we have constants in the DB library to represent these key lengths ?

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please address the comments

File: app/config/collections.php (hunk 3 above)
Comment (fogelito) [reply app/config/collections.php:2427]: That's is true 
I will have to make another tag for this

File: app/config/collections.php (hunk 2 above)
Comment (fogelito) [reply app/config/collections.php:2079]: I consult this with Torsten, we can change it up to 768 without any problems, If we need more I will have to restrict the index length

**Review by eldadfux — APPROVED** [review]

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**